src/js/api/core-shim.js

```diff
@@ -78,7 +78,8 @@ Object.assign(CoreShim.prototype, {
 
         // Assigning config properties to the model needs to be synchronous for chained get API methods
         const configuration = Config(options, persisted);
-        Object.assign(model.attributes, configuration, INITIAL_PLAYER_STATE);
+        const setupConfig = { setupConfig: Object.assign({}, options) };
```

Comment:
You should do this in "Config" before `options` has properties appended to it. Then there will be no changes to core-shim.js and you can assign  `setupConfig` directly:

```js
// In config.js after let config = Object.assign({}, Defaults, allOptions); (should be const)
config.setupConfig = Object.assign({}, options);
```

Reply:
@robwalch  Making the changes now.. It will still have the ID appended to it, which occurs in the setup method in api.js, as well as the analytics block, which is added before that.